### ISSUE TITLE
Question: Does App Name Have to be Unique?

### ISSUE THREAD

**Issue by pkscout** [ChloeCodesThings/FakeCallWorkshop#3]
When creating the app name, does that have to be unique across every Azure instance (given the URL scheme I think it does)? If so, you might want to note that so that everyone doesn't try and use the name included in the example screenshot.  For instance, I used my initials plus a dash and then fakebfworkshop.

**Comment by gethari** [comment]
Yes, FunctionApp names should be unique globally. You cannot create an app with the same name that you see on any repo, coz that person might have already used it
